Packs/Base/Scripts/GetErrors/GetErrors.py

```diff
@@ -0,0 +1,44 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def get_errors(entries):
+    """
+        entries should be a list of demisto entries
+
+        :type entries: ``list``
+        :param entries: multiples entries of results of demisto.executeCommand()
+
+        :return: Error message extracted from the demisto.executeCommand() result
+        :rtype: ``string``
+    """
+    error_messages = []
+    for entry in entries:
```

Comment:
you can use `is_error` and `get_error` functions from CSP.

Reply:
No, I can't - needed the logic a bit different.